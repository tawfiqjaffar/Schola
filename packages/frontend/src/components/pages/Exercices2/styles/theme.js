import gray from "@material-ui/core/colors/grey";
import lightGreen from "@material-ui/core/colors/lightGreen";

import { createMuiTheme } from "@material-ui/core/styles";

const theme = createMuiTheme({
    palette: {
        primary: {
            light: lightGreen[200],
            main: lightGreen[400],
            dark: lightGreen[600],
            contrastText: "#fff"
        },
        secondary: {
            light: gray[200],
            main: gray[500],
            dark: gray[700],
            contrastText: "#fff"
        }
    }
});

export default theme;
