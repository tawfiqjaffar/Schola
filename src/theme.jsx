import { createMuiTheme } from '@material-ui/core/styles';

const palette = {
  primary: { main: '#70C6C7', contrastText: '#212121' },
  secondary: { main: '#60c07a', contrastText: '#FAFAFA' },
  text: {
    primary: '#212121',
    secondary: '#FAFAFA',
  },
  background: {
    paper: '#70C6C7',
  },
};
const themeName = 'Downy Pastel Green Gayal';

export default createMuiTheme({ palette, themeName });
