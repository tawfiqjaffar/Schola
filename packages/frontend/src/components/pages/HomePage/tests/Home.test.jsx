import React from 'react';

import renderer from 'react-test-renderer';

import { ThemeProvider } from '@material-ui/core';
import { BrowserRouter } from 'react-router-dom';
import Home from '../HomePage';
import theme from '../../../../theme';

describe('Home', () => {
  it('should render Home Page', () => {
    const component = renderer.create(
      <ThemeProvider theme={theme}>
        <BrowserRouter>
          <Home />
        </BrowserRouter>
      </ThemeProvider>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
