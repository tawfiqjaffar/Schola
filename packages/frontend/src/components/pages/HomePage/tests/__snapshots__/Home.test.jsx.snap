// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render Home Page 1`] = `
<div
  className="MuiContainer-root makeStyles-root-1 MuiContainer-maxWidthLg"
>
  <div
    className="MuiPaper-root makeStyles-paper-2 MuiPaper-elevation1 MuiPaper-rounded"
  >
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3"
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4 MuiGrid-grid-lg-3"
      >
        <div
          className="MuiPaper-root makeStyles-paper-2 makeStyles-fixedHeight-3 makeStyles-background-4 MuiPaper-elevation1 MuiPaper-rounded"
        >
          Profil_img
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4 MuiGrid-grid-lg-6"
      >
        <div
          className="MuiPaper-root makeStyles-paper-2 makeStyles-fixedHeight-3 makeStyles-background-4 MuiPaper-elevation1 MuiPaper-rounded"
        >
          Infos_etudiant
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4 MuiGrid-grid-lg-3"
      >
        <div
          className="MuiPaper-root makeStyles-paper-2 makeStyles-fixedHeight-3 makeStyles-background-4 MuiPaper-elevation1 MuiPaper-rounded"
        >
          graph_moyennes
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4 MuiGrid-grid-lg-9"
      >
        <div
          className="MuiPaper-root makeStyles-paper-2 makeStyles-fixedHeight-3 makeStyles-background-4 MuiPaper-elevation1 MuiPaper-rounded"
        >
          ?
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-4 MuiGrid-grid-lg-3"
      >
        <div
          className="MuiPaper-root makeStyles-paper-2 makeStyles-fixedHeight-3 makeStyles-background-4 MuiPaper-elevation1 MuiPaper-rounded"
        >
          notifications
        </div>
      </div>
    </div>
  </div>
</div>
`;
